deployments/AKS/kubearmor.yaml

```diff
@@ -56,6 +56,13 @@ spec:
         name: kubearmor-relay-server
         ports:
         - containerPort: 32767
+        resources:
+          limits:
+            cpu: "4"
+            memory: 4Gi
+          requests:
+            cpu: 100m
+            memory: 512Mi
```

Comment:
A question - @achrefbensaad mentioned that it will be a waste of resource to use 4 CPUs while requesting 100m, have we verified using lesser CPU?